src/Spec2-Layout/SpExecutableLayout.class.st

```diff
@@ -15,6 +15,18 @@ SpExecutableLayout >> adapterName [
 	^ self subclassResponsibility
 ]
 
+{ #category : #'as yet unclassified' }
+SpExecutableLayout >> add: aPresenter height: anInteger [ 
```

Comment:
Those methods should probably be on SpBoxLayout only since the other layout constraints don't have #height: and #width: